lib/livebook_web/live/session_live.ex

```diff
@@ -2169,7 +2169,9 @@ defmodule LivebookWeb.SessionLive do
       section_views: section_views(data.notebook.sections, data),
       bin_entries: data.bin_entries,
       secrets: data.secrets,
-      hub: Livebook.Hubs.fetch_hub!(data.notebook.hub_id),
+      hub:
+        Livebook.Hubs.get_offline_hub(data.notebook.hub_id) ||
+          Livebook.Hubs.fetch_hub!(data.notebook.hub_id),
```

Comment:
Should we account for the offline hub in `fetch_hub!` or have a separate function? It's the second place we do this and I can imagine we forget somewhere in the future :)

Reply:
That's what the PR before did and the thing is that we want to ignore the offline hub of 99% of the cases. It is only used to validate the stamp and nothing more. :)

Reply:
Actually, `data.notebook.hub_id` is always going to be personal hub, because we ignore `:hub_id` on import (in both places).

Reply:
I am not sure what you mean but feel free to figure out the details with Ale. The goal is to validate the stamp with the offline hub.

Reply:
As per the other comment, with `Hubs.get_offline_hub(hub_id) -> {attrs, messages}`, we effectively ignore `hub_id` in the .livemd file and so we will try to verify the stamp with the default hub (personal). I agree that we need a test :D

Reply:
I see, I understand. Maybe we need to break that code in two parts: one with the hub id and another with the stamp_hub_id. :)

Reply:
Sounds good! @aleDsz so when importing metadata we can return `{attrs, stamp_hub_id, messages}` and then use that `stamp_hub_id` in `postprocess_stamp`.

Reply:
And as per this line here, I don't think we need `get_offline_hub`, because `notebook.hub_id` is not supposed to point to the offline hub :)